dls-controls/malcolmjs
# Feature/nav bar #28

## Description

Refactored the nav bar structure out of the drawer container and into a separate component.
Updated the nav bar so it takes the part of the current route based on blocks and views.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run `npm start` and navigate to http://localhost:3000/gui/PANDA//PANDA:SEQ1 and observe the nav bar in the top middle of the window.

## Agile board tracking

connect to #28
